res/values/resurrection_device_maintainers_strings.xml

```diff
@@ -592,6 +592,9 @@
   <string name="device_land">Xiaomi Redmi 3s</string>
   <string name="device_land_summary">land\nMaintainer - Saicharan </string>
 
+  <string name="device_santoni">Xiaomi Redmi 4X</string>
+  <string name="device_santoni_summary">santoni\nMaintainer - IsaacChen </string>
```

Comment:
I searched and couldn't find Official XDA page for this official dev. Is forum to be created?